requirements.txt

```diff
@@ -7,7 +7,6 @@ scipy
 paramiko
 pysftp
 sox
-python_speech_features
```

Comment:
I think we should leave this here during the deprecation period, that way it works out of the box even if libdeepspeech isn't installed.